core/templates/third-party-imports/guppy.import.ts

```diff
@@ -16,5 +16,5 @@
  * @fileoverview This file imports the Guppy library.
  */
 
-(window as Window).Guppy = require(
-  'static/guppy-f509e1/build/guppy.min.js');
+(window as Window).Guppy = require('guppy-dev/build/guppy.min.js');
+import 'guppy-dev/build/guppy-default.min.css';
```

Comment:
Since you are already importing it here why the css file linked in the html file above?

Reply:
yes, not needed above